src/main/java/org/hawkular/client/android/backend/BackendClient.java

```diff
@@ -130,6 +128,13 @@ public void getTenants(@NonNull Activity activity, @NonNull Callback<List<Tenant
                 BackendPipes.Paths.TENANTS), callback);
     }
 
+    public void getEnvironments(@NonNull Tenant tenant,
+                                @NonNull Activity activity, @NonNull Callback<List<Environment>> callback) {
+        PipeManager.getPipe(BackendPipes.Names.ENVIRONMENTS, activity)
+            .read(getFilter(
```

Comment:
Should we change it for [read(String id, Callback<T> callback)](https://github.com/aerogear/aerogear-android-pipe/blob/master/aerogear-android-pipe/src/main/java/org/jboss/aerogear/android/pipe/Pipe.java#L40-L46)?